server/test/support/grpc_integration_case.ex

```diff
@@ -0,0 +1,59 @@
+defmodule BuffServerGrpc.IntegrationCase do
+  @moduledoc false
```

Comment:
so using the IntegrationCase will setup a GRPC server for u, very cool

Reply:
it defined functions to get the stuff you need

Reply:
Adding comments.